PUGTeologia.csl

```diff
@@ -0,0 +1,509 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" initialize-with-hyphen="true" page-range-format="expanded" demote-non-dropping-particle="sort-only" name-delimiter=" – " names-delimiter=" – " delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" name-form="long" initialize-with=". " sort-separator=", ">
+  <info>
+    <title>PUGTeologia</title>
+    <title-short>PUGTeologia</title-short>
+    <id>http://www.zotero.org/styles/PUGTeologia</id>
+    <link href="http://www.zotero.org/styles/PUGTeologia" rel="self"/>
+    <link href="http://www.unigre.it" rel="documentation"/>
```

Comment:
Please give a link to the actual style guidelines, not just the university webpage